### DISCUSSION TITLE
Question regarding 5th homework

### DISCUSSION THREAD

**Discussion by CS-MLU** [cafawo/FinancialDataAnalytics#13] [Q&A]
In the current homework we are tasked with showing the tracking error in the graphs as well. How are we to show the tracking error since it is just one value?

**Comment by cafawo** [comment]
Tracking error "indicates how closely a portfolio follows the index to which it is benchmarked" [see HERE](https://en.wikipedia.org/wiki/Tracking_error). It can therefore also be expressed as a time series, i.e. the difference between benchmark and portfolio.
